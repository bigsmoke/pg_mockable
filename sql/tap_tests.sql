begin transaction;

create extension pg_mockable version '0.1.8' cascade;

call mockable.test__pg_mockable();

rollback transaction and chain;

create extension pg_mockable cascade;

call mockable.test__pg_mockable();

rollback transaction;
