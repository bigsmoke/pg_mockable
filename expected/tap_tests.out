begin transaction;
create extension pg_mockable
    cascade;
NOTICE:  installing required extension "hstore"
call mockable.test__pg_mockable();
rollback transaction;
